react/InstitutionalContext.js

```diff
@@ -0,0 +1,4 @@
+// This context simplifies the check for institutional pages
```

Comment:
What do you mean? How does this help?

Reply:
A context is provided for every custom route. It can be (and it is) used to filter different kinds of routes, like product pages, category pages, insitutional pages, etc.

A simplified route object:
```
context: "vtex.store@2.x/InstitutionalContext"
path: "/extra-page"
routeId: "vtex.store@2.x:store.institutional#extra-page"
title: "Extra"
```

Reply:
Wouldn't checking for `routeId.indexOf('vtex.store@2.x:store.institutional') === 0` be better?

Reply:
Sounds like a hack to add this Context just for that

Reply:
I personally rather use context for that. Our first attempt to filter institutional pages was something like you suggested, but we are using context for all of the other kinds of pages, and is not a good approach to check for apps version like `vtex.store@2x...`